gradle.properties

```diff
@@ -1,9 +1,9 @@
-dropwizardVersion=1.3.5
+dropwizardVersion=1.0.9
```

Comment:
Appears my release merging months ago broke the 1.0.x branch.  DW Version 1.3.x does not help when you need a 1.0.x project.  Looking at merge history this appeared to be the primary change.